index.html

```diff
@@ -1412,6 +1412,123 @@ <h3>Biometrics</h3>
     </section>
   </section>
 
+  <section class="informative">
+    <h2>Environmental and Ethical Considerations</h2>
+
+    <p>
+      The following section details certain key areas of focus when implementing a 
+      new DID menthod, or implementing a solution that incorporates DIDs.
+    </p>
+
+    <p>
+      A reliable guide for making assessments of various technologies and weighing
+      ethical considerations is the 
+      <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG Ethical Web Principles</a>
+      document.
+    </p>
+
+    <section class="informative">
+      <h3>Broad Ethical Principles</h3>
+      <p>
+        As with all things, strong consideration for the appropriate and ethical use
+        of technology should be made when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">Principles</a>
+        section of the Ethical Web Principles, there are certain key goals that should 
+        apply to all Web standards and technologies. DIDs explicitly support several
+        of these goals, especially the following:
+        <ul>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">Privacy</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">Individual Control</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">Device Independence</a></li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Energy Usage and Environmental Impacts</h3>
```

Comment:
We should consider reframing this section to address the issue from an empirical perspective, for example:

```suggestion
      <h3>Decentralized System Security Considerations</h3>
```

Reply:
Agreed - I think the trick is to fully acknowledge some of the concerns from folks looking at DIDs, but show clearly where we think there are tradeoffs that merit a developer picking one approach vs another.  e.g. DIDs engaged in use cases related to requirements for strong personal privacy and control vs other cases

Going to be taking a pass on a rewrite of this whole section today

Reply:
Showing tradeoffs that YOU THINK merit one thing or another is inappropriate for guiding implementations, and is appropriate for the Rubric work.

Reply:
Referring to the Rubric -- including its developing metrics/focuses regarding energy consumption *and* security features *and* other features, and the relations between these (which may include notes like "barring special attention, a change in the level of this benefit/cost will typically cause a parallel/inverse/multiple change in the level of that benefit/cost") -- all of which will play a role in deployment choices a/k/a method adoption -- *is* appropriate for the Implementation Guide.

Reply:
"Securing Decentralized Identifier Systems" is defined in the spec as https://www.w3.org/TR/did-core/#dfn-verifiable-data-registry and we should avoid inventing new words for the same concept.